onchain/rollups/test/foundry/consensus/authority/AuthorityHistoryPairFactory.t.sol

```diff
@@ -148,11 +170,11 @@ contract AuthorityFactoryTest is Test {
             ) {
                 ++numOfNewHistory;
 
-                address a;
+                NewHistoryEventData memory eventData;
 
-                a = abi.decode(entry.data, (address));
+                eventData = abi.decode(entry.data, (NewHistoryEventData));
```

Comment:
These 2 versions are essentially the same, right? I think we don't have to wrap a struct for every `abi.decode`. Wrapping a struct makes a lot of sense when there are multiple (different) data types. But when there's only 1 (or maybe 2) data type, I don't think we should wrap it. 
Regarding readability, this line could become:
`address history =  abi.decode(entry.data, (address));`

Reply:
Yes, both versions are isomorphic. We could decode the data into address directly, but if we are using this struct abstraction either way, we might as well be consistent with it, regardless of the number of fields.

Reply:
I'll keep it simple, I see the importance of being consistent but it's no sense to use a struct for a simple address decoding.

Reply:
Fair enough. :-)